src/database/DataTypes/BreachLog.js

```diff
@@ -0,0 +1,21 @@
+/**
+ * Sustainable Solutions (NZ) Ltd. 2020
+ * mSupply Mobile
+ */
+
+import Realm from 'realm';
+
+export class BreachLog extends Realm.Object {}
+
+BreachLog.schema = {
```

Comment:
maybe .. `Breach` or `TemperatureBreach`.. or `TemperatureBreachLog`...?

Reply:
🤔 `TemperatureBreach` for me - because *Breach* is fairly general and needs a bit of context, and the class is tied to temperatures because of the `temperatureLogs` property, and this class is an object or an entry - a collection of them could constitute a log. This class could be 'logEntry'

that's just my 2c though..